src/expressjs/controllers/products.js

```diff
@@ -0,0 +1,33 @@
+import * as products from '../config/products';
+import fs from 'fs';
+import path from 'path';
+
+export const getAllProducts = (req, res) => {
+    return res.json(products.default);
+}
+
+export const getProduct = (req, res) => {
+    const { id } = req.params;
+    const result = products.default.find(product => product.id === parseInt(id));
```

Comment:
Great that you're using `parseInt` instead of implicit `+`, but there's one tip you should always show the radix of your int, like `parseInt(id, 10)` because sometimes there may be inconsistencies in engines and your int may become not of 10 radix, but for example 8

Reply:
Thanks, updated code.